examples/js/ShaderSkin.js

```diff
@@ -167,7 +167,7 @@ THREE.ShaderSkin = {
 				"diffuseColor = diffuseColor * colDiffuse;",
 
 				"vec3 normal = normalize( vNormal );",
-				"vec3 viewPosition = normalize( vViewPosition );",
+				"vec3 viewerDirection = normalize( vViewPosition );",
```

Comment:
> Does the inconsistent specular computation for hemispherical light sources in this file actually make sense?

Well, hemisphere lights are a model for ambient light, and typically do not have a specular contribution of that sort. My intention is to remove the specular terms from hemisphere lights in `MeshPhongMaterial` and make sure hemi-lights are property attenuated by AO maps.